rsfbclient-native/src/row.rs

```diff
@@ -124,12 +142,75 @@ impl ColumnBuffer {
             Float => ColumnType::Float(float_from_buffer(&self.buffer)?),
 
             Timestamp => ColumnType::Timestamp(timestamp_from_buffer(&self.buffer)?),
+
+            BlobText => ColumnType::Text(blobtext_to_string(&self.buffer, db, tr, ibase)?),
         };
 
         Ok(Column(Some(col_type)))
     }
 }
 
+/// Converts a text blob to a string
+fn blobtext_to_string(
+    buffer: &[u8],
+    db: &mut ibase::isc_db_handle,
+    tr: &mut ibase::isc_tr_handle,
+    ibase: &IBase,
+) -> Result<String, FbError> {
+    let mut final_string = String::new();
+
+    let mut status = Status::default();
+    let mut handle = 0;
+
+    let (head, body, _) = unsafe { buffer.align_to::<ibase::GDS_QUAD_t>() };
```

Comment:
I think something on the lines of the `timestamp_from_buffer` can be used here to avoid the unsafe and assertion.

Related code:
```rust
let date = ibase::ISC_TIMESTAMP {
    timestamp_date: ibase::ISC_DATE::from_ne_bytes(buffer[0..4].try_into().unwrap()),
    timestamp_time: ibase::ISC_TIME::from_ne_bytes(buffer[4..8].try_into().unwrap()),
};
```

Reply:
Done!